tests/test_client/tests.py

```diff
@@ -590,6 +597,21 @@ def test_session_modifying_view(self):
         # Check that the session was modified
         self.assertEqual(self.client.session['tobacconist'], 'hovercraft')
 
+    @override_settings(
+        INSTALLED_APPS=[],
+        SESSION_ENGINE='django.contrib.sessions.backends.file',
+    )
+    def test_sessions_app_is_not_installed(self):
+        self.test_session_modifying_view()
+
+    @override_settings(
+        INSTALLED_APPS=[],
+        SESSION_ENGINE='django.contrib.sessions.backends.nonexistent',
+    )
+    def test_session_engine_is_invalid(self):
+        with self.assertRaises(ImportError):
```

Comment:
To verify this is the expected import error, I'd do something like:
`self.assertRaisesMessage(ImportError, 'nonexistent')`